core/trino-main/src/main/java/io/trino/operator/VariableWidthData.java

```diff
@@ -70,14 +74,17 @@ public VariableWidthData(List<byte[]> chunks, int openChunkOffset)
     {
         this.chunks.addAll(chunks);
         this.openChunkOffset = openChunkOffset;
-        this.chunksRetainedSizeInBytes = chunks.stream().mapToLong(SizeOf::sizeOf).sum();
+        this.chunksRetainedSizeInBytes = chunks.stream().mapToLong(SizeOf::sizeOf).collect(() -> 0L, Math::addExact, Math::addExact);
```

Comment:
This needs to be
```java
.reduce(0, Math::addExact);
```
`.collect()` is a mutable reduction, so it's expecting that `addExact()` is mutating the state. IntelliJ will warn that the result is unused (since the lambda interface is a consumer).